docs/user/explanation/dedicated-platform.rst

```diff
@@ -0,0 +1,317 @@
+.. This page seeks to put out lots of pointers to other articles in the documentation
+.. while giving an introduction that can be read consecutively.
+.. TODO:
+.. - Links and references!
+.. - Diagram in life-cycle section
+..
+.. There are other reasons that we could cover:
+.. - Code: If you write docs as code, you want this
+.. - When you should NOT choose a dedicated documentation CI/CD: You just need a README in your Git repo! You need WYSIWYG so badly that you're probably better off with Confluence, GitBook or Google Docs.
+.. - A dedicated platform helps to solve tasks that you otherwise end up writing and maintaining your own scripts and CI tools for.
+.. - Always evolving: Read the Docs develops new features on-demand. So you get a dedicated platform that's maintained and has new features added.
+.. - Community: Related to the above, but perhaps worth mentioning that a lot of people are building tools and extensions that run on the platform etc.
+.. - Reader features: Alternative name for "Batteries included" is "Reader features"
+
+
+Choosing a dedicated documentation platform
+===========================================
+
+This is an introduction to the most common benefits of choosing a *dedicated* platform for building and publishing documentation projects.
+
+Let us start with a high-level overview:
+
+🧭️️️ :ref:`lifecycle`
+    A dedicated documentation platform has all the features to handle a variety of challenges and complexities that documentation projects meet during their lifecycle.
+
+🛠️ :ref:`documentation_tools`
+    By choosing a tool-agnostic dedicated platform,
+    you can use documentation tools from open source eco-systems.
+
+🚢️️ :ref:`workflows`
+    A dedicated documentation platform works like a :term:`CI/CD platform <CI/CD>`, publishing and versioning your documentation automatically.
+    Most documentation projects need CI/CD automation,
+    and by choosing a dedicated platform,
+    you save building your own scripts and deployment workflows for documentation.
+
+🔋️ :ref:`batteries_included`
+    Read the Docs continues to develop new projects and ideas,
+    bringing additional powers to documentation projects that are hosted on the platform.
+
+.. The following sections are about using a dedicated platform as a first-choice in order to free up space for other tasks.
+.. These paragraphs can benefit from being shortened and organized, but they are intended as the summary of the introduction.
+
+Using Read the Docs
+allows you to focus on other critical tasks,
+such as choosing documentation tools, structuring your documentation and having the perfect workflow.
```

Comment:
"Structuring your documentation" could highlight/mention the page where we talk about Diataxis that I think we already wrote.

Reply:
Yes, it definitely should. This content was just recently broken off into a separate PR to make this one lighter :)

Reply:
Added a note :)